field/age/classes/field.php

```diff
@@ -712,19 +712,16 @@ public function userform_set_prefill($fromdb) {
      * @return string - the string for the export file
      */
     public function userform_db_to_export($answer, $format='') {
+        // The content of the provided answer.
         $content = $answer->content;
-        if ($content == SURVEYPRO_NOANSWERVALUE) { // Answer was "no answer".
-            return get_string('answerisnoanswer', 'mod_surveypro');
-        }
-        if ($content == SURVEYPRO_ANSWERNOTINDBVALUE) { // Item was disabled. (Used by frequenct report).
-            return get_string('notanswereditem', 'mod_surveypro');
-        }
-        if ($content === null) { // Item was disabled.
-            return get_string('notanswereditem', 'mod_surveypro');
+        $parentcontent = parent::userform_db_to_export($answer, $format);
+        if ($parentcontent != $content) {
```

Comment:
What does this mean? In which cases $parentcontent won't return the content to be used? I don't understand it, specially if the next line means that $parentcontent always "win". Then you don't need the condition at all.